### DISCUSSION TITLE
每次添加新数据都需要重新加载吗

### DISCUSSION THREAD

**Discussion by rilweic** [milvus-io/milvus#18110] [Q&A and General discussion]
Milvus 做图片搜索，每次有新的图片的时候是否需要重新执行locaCollection操作啊

**Comment by yhmo** [comment]
load是调用一次即可，新增的数据会被自动加载，如果某个segment建好索引后，其索引会被自动加载并替换掉缓存中原先的原始向量。

PS: 鼓励用英文提问哈
